doc/examle_ospd.conf

```
@@ -0,0 +1,31 @@
+[OSPD - openvas]
+## General
+pid_file = install-prefix/var/run/ospd/openvas.pid
+lock_file_dir = install-prefix/var/run/
+stream_timeout = 1
+max_scans = 3
+min_free_mem_scan_queue = 1000
+max_queued_scans = 0
+
+# Log config
+log_level = DEBUG
+log_file = install-prefix/var/log/gvm/openvas.log
+log_config = install-prefix/.config/ospd_log.conf
```

I would suggest to use minus instead of underscore for file names. The other files are using minus too.
```suggestion
log_config = install-prefix/.config/ospd-logging.conf
```